service/src/main/java/bio/terra/workspace/service/workspace/flight/CheckSpendProfileStep.java

```diff
@@ -1,5 +1,8 @@
 package bio.terra.workspace.service.workspace.flight;
 
+import static bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys.AZURE_BILLING_MANAGED_RESOURCE_GROUP_ID;
```

Comment:
I think we should pass the whole SpendProfile object instead of each part.
That might require changing SpendProfile.java to a record so serdes will work.